src/Microsoft.Extensions.DependencyModel/CompilationLibrary.cs

```diff
@@ -51,31 +51,31 @@ public class CompilationLibrary : Library
 
         public IEnumerable<string> ResolveReferencePaths()
         {
-            return ResolveReferencePaths(DefaultResolver);
+            var assemblies = new List<string>();
+
+            return ResolveReferencePaths(DefaultResolver, assemblies);
         }
 
-        public IEnumerable<string> ResolveReferencePaths(params ICompilationAssemblyResolver[] additionalResolvers)
+        public IEnumerable<string> ResolveReferencePaths(params ICompilationAssemblyResolver[] customResolvers)
```

Comment:
I also don't like exposing DefaultResolver outside of the class (either explicitly or implicitly by passing references to it outside of the class).